example_scenes/manim.cfg

```diff
@@ -0,0 +1,12 @@
+[log.color]
+logging.keyword = magenta
```

Comment:
Please add a comment on this file that says that these colors are to be parsed by `rich` (or by colour?), and NOT the same colors that manim defines in `constants.py`.